refinery/core/models.py

```diff
@@ -1928,6 +1957,12 @@ def __unicode__(self):
             self.name + " (" + str(self.is_refinery_file) + ")"
         )
 
+    def get_input_connection_id(self):
+        return "{}_{}".format(self.step, self.filename)
+
+    def get_output_connection_id(self):
+        return "{}_{}".format(self.step, self.name)
+
```

Comment:
oy: I hadn't noticed this discrepancy. Maybe a comment to explain why filename for one and name for the other?